ratis-examples/src/main/resources/conf.properties

```diff
@@ -15,4 +15,5 @@
 # limitations under the License.
 
 raft.server.address.list=127.0.0.1:10024,127.0.0.1:10124,127.0.0.1:11124
-# raft.server.root.storage.path
\ No newline at end of file
+# raft.server.root.storage.path
+# raft.server.priority.list=1,0,0
```

Comment:
How about adding slowness list here instead of passing it in CLI?
```
# raft.server.simulated-slowness.list=0,1s,0
```

Reply:
It's a great idea! Saved me many nasty parsing code.